package.json

```diff
@@ -264,7 +264,7 @@
     "blob-polyfill": "^2.0.20171115",
     "bootstrap": "4.3.1",
     "bootswatch": "4.3.1",
-    "cesium": "1.49.0",
+    "cesium": "1.56.1",
```

Comment:
The Cesium upgrade has some additions like `Scene.highDynamicRange` that change how the globe is rendered fairly noticeably. I'm wondering if we should figure out what all changed and adjust to look more like it did previously. See differences here:

<img width="772" alt="Screen Shot 2019-04-23 at 8 43 49 AM" src="https://user-images.githubusercontent.com/5685437/56590433-55cc3900-65a4-11e9-840a-9391a6c8400b.png">